runtime/list_test.go

```diff
@@ -353,6 +353,30 @@ func TestListIteratorIter(t *testing.T) {
 	}
 }
 
+func TestListPop(t *testing.T) {
+	pop := mustNotRaise(GetAttr(NewRootFrame(), ListType.ToObject(), NewStr("pop"), nil))
+	fun := wrapFuncForTest(func(f *Frame, l *List, args ...*Object) (*Tuple, *BaseException) {
+		result, raised := pop.Call(f, append(Args{l.ToObject()}, args...), nil)
+		if raised != nil {
+			return nil, raised
+		}
+		return newTestTuple(result, l), nil
+	})
+	cases := []invokeTestCase{
+		{args: wrapArgs(newTestList(-1, 0, 1)), want: newTestTuple(NewInt(1).ToObject(), newTestList(-1, 0).ToObject()).ToObject()},
```

Comment:
You can pass primitive Go types into newTestTuple (and newTestList, newTestDict, wrapArgs, etc.) Also, Grumpy types like *List, *Tuple, etc. don't need to be converted to object to be passed in. This can be:

`newTestTuple(1, newTestList(-1, 0)).ToObject()`

Can make similar simplifications below as well.